packages/icons-cli/src/svg/optimizeSvg.ts

```diff
@@ -16,8 +16,25 @@ const svgo = new Svgo({
   ],
 })
 
-export async function optimizeSvg(input: string, convertedColor: string) {
-  const { data } = await svgo.optimize(input)
+/**
+ * SVGを最適化するオプション
+ */
+interface Options {
+  /**
+   * currentColorに置換する色 #ffffff
+   */
+  convertedColor: string
+  /**
+   * svgoによる最適化を行わない
+   */
+  withoutOptimizeBySVGO?: boolean
+}
+
+export async function optimizeSvg(input: string, options: Options) {
```

Comment:
[MAY] これ系はデフォルト値を与えたいかな

```suggestion
export async function optimizeSvg(input: string, { convertedColor, withoutOptimizeBySVGO = false }: Options) {
```